numeric.rb

```diff
@@ -233,11 +233,14 @@ def zero?
 #    Integer.try_convert(object) -> object, integer, or nil
 #
 #  If +object+ is an \Integer object, returns +object+.
+#    Integer.try_convert(1) # => 1
 #
 #  Otherwise if +object+ responds to <tt>:to_int</tt>,
 #  calls <tt>object.to_int</tt> and returns the result.
+#    Integer.try_convert("1") # => 1
```

Comment:
`String#to_int` is not defined by default.

Reply:
Do you think an example with `Float` would be better? Any other suggestions?

Reply:
```suggestion
#    Integer.try_convert(1.25) # => 1
```